tests/app.yaml

```diff
@@ -351,3 +351,26 @@ cp2k:
       result:
         regex:  'CP2K\s+1\s+1.0'
         action: store_true
+
+#-------------------------------------------------------------------
+# Trycycler Test
+#-------------------------------------------------------------------
+trycycler:
+  summary: Trycycler test
+  build:
+    cmds:
+      - wget https://github.com/rrwick/Trycycler/archive/v0.5.3.tar.gz
```

Comment:
This will download these files everytime the test is run. Instead, create a tarball for the test that includes the data files and place it in the `test_src` directory. Then specify the name of the tarball in the `source_path:`. See other tests for examples.

Reply:
The files are downloaded the first time the test is run, then a link is made to the downloaded files
e.g.

```
vortex1$ cd ~/testsuite/working_dir/
vortex1$ find . -name v0.5.3.tar.gz -exec ls -l {} \;
-r--r--r-- 1 tonykew tonykew 3147707 Jan 25 12:41 ./builds/f26b5a4fb8405e48/v0.5.3.tar.gz
lrwxrwxrwx 1 tonykew tonykew 73 Jan 25 12:41 ./test_runs/40/build/v0.5.3.tar.gz -> /home/tonykew/testsuite/working_dir/builds/f26b5a4fb8405e48/v0.5.3.tar.gz
lrwxrwxrwx 1 tonykew tonykew 73 Jan 25 12:45 ./test_runs/41/build/v0.5.3.tar.gz -> /home/tonykew/testsuite/working_dir/builds/f26b5a4fb8405e48/v0.5.3.tar.gz
vortex1$ 
```